.vscode/launch.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
I intentionally added this file to ease vs-code development.

Reply:
Ah got it, makes sense

Reply:
I'm a +1 on removing this, we don't really need to support particular editors when working on this as it can set a precedent for needing  to support other editors in here (such as vim, goland, emacs, sublime text, etc..)